### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by Gleennkar**
## My First Code Review
- Added an index.html
- Added  style.css
- updated my README.md
- Survey Form from Codeacademy

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>FCC: Survey Form</title>
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
```
Comment (Gleennkar) [inline index.html:12]: Proper use of html semantics like the header section will be helpful here, this makes it easier to navigate your code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
```
Comment (Gleennkar) [inline index.html:13]: Proper use of html semantics like the header section will be helpful here, this makes it easier to navigate your code

File: style.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    width: 600px;
+    padding: 10px;
+    border-radius: 10px;
+}
+#number{
+    width: 600px;
+    padding: 10px;
+    border-radius: 10px;
+}
+#dropdown{
```
Comment (Gleennkar) [inline style.css:41]: Try to give your form element classes. This will make styling your form much easier and it will look presentable

**Review by Gleennkar — COMMENTED** [review]
Hi Gleeny🙂 ,

You have done a tremendous job setting up your project. I really like it 👍
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights
Didnt run linter checks
No Correct use of GitHub flow 
Not well-documented Repository 
 Code base not  very clean and readable

Required changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by sumairq — APPROVED** [review]
### Hi @Gleennkar  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Great Communication
✔️ Friendly tone
✔️Made good inline comments with explanations 🥇
✔️Awesome review summary

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**